templates/server/puppetserver/conf.d/auth.conf.erb

```diff
@@ -12,7 +12,13 @@ authorization: {
                 type: regex
                 method: [get, post]
             }
-            allow: <%= @server_trusted_agents + ['$1'] %>
+<%= scope.call_function(
+  'to_json_pretty',
+  [
+    {'allow' => @server_trusted_agents + ['$1'] + @server_trusted_certificate_extensions.map { |extension| { 'extensions' => extension } } }
+  ]
+).lines.to_a[1..-2].map{ |line| "          #{line}"}.join
+-%>
```

Comment:
We migrated away from puppetlabs-hocon because puppetlabs-hocon because back then the support wasn't really clear. Looking at how they still update it I do wonder if we should reconsider. However, probably better to merge it now.

Reply:
I think it'd be much easier for users to add their own configs in their profiles if it still did use puppetlabs-hocon